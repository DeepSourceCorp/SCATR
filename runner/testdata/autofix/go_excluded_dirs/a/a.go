package a

func main() {
	a := 10
	a = a
	_ = a
}
