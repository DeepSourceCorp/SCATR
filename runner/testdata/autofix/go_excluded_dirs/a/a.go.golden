package a

func main() {
	a := 10
	_ = a
}
